lib/controller.js

```diff
@@ -66,11 +66,11 @@ controller.setupLocal = function(opts) {
 
 
 Tessel.list = function(opts) {
-
   return new Promise(function(resolve, reject) {
-
+    log.spinner.stop();
```

Comment:
Is this just making sure the spinner is defaulted to stop before starting it, in case it was not stopped by `callControllerWith()`?